pallets/remote-asset-manager/src/mock.rs

```diff
@@ -442,7 +458,7 @@ pub mod para {
         type MinimumRemoteStashBalance = MinimumRemoteStashBalance;
         type Assets = Currency;
         type XcmExecutor = XcmExecutor<XcmConfig>;
-        type XcmAssets = xcm_assets::XcmAssetExecutor<XcmAssetConfig>;
+        type XcmAssetTransfer = XTokens;
```

Comment:
So, if we use `xToken` directly, we don't to convert the errors anymore

Reply:
yeah this is now outsourced to xtokens.